src/Symfony/Component/Security/Core/Authorization/Voter/Voter.php

```diff
@@ -43,17 +43,59 @@ public function vote(TokenInterface $token, mixed $subject, array $attributes):
             }
 
             // as soon as at least one attribute is supported, default is to deny access
-            $vote = self::ACCESS_DENIED;
+            $vote = $this->deny();
```

Comment:
`$vote` is reset at each loop, IMO is not ok
What do you think of storing Vote messages as array internally and method getMessage will implode them ?

Reply:
Indeed, could be great like form validations errors

Reply:
Now string and string array are supported.

(Insipred by Groups annotation)